azerothcore/azerothcore-wotlk
# fix(Scripts/AzjolNerub): Amanitar not casting “Mini” and mushroom not cleansing the players debuff on death

## CHANGES PROPOSED:
-  Make shrooms remove the mini debuf for the killing player
- Add an event to the boss to cast mini every 30 seconds
References: https://www.youtube.com/watch?v=j--kwwjgV-0 and https://www.youtube.com/watch?v=d3gHO7I8A-o


## ISSUES ADDRESSED:
Closes https://github.com/azerothcore/azerothcore-wotlk/issues/3344



## TESTS PERFORMED:
- Started boss, and testes mini casting timing and removing it with the shrooms.



## HOW TO TEST THE CHANGES:

1. Set dung to HC
1.1. .cheat god or .mod hp 100000000 :)
2. .go c 202410
3. Start boss
4. Confirm that after 30 seconds he will cast mini
5. After boss casts mini kill healthy shroom to  remove the mini debuff


## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
